tests/integration/test_core.py

```diff
@@ -133,3 +109,78 @@ async def test_wordpress_functionality(
             admin_username="admin",
             admin_password=default_admin_password
         )
+
+
+@pytest.mark.asyncio
+async def test_wordpress_default_themes(
+        unit_ip_list,
+        get_theme_list_from_ip
+):
+    """
+    arrange: after WordPress charm has been deployed and db relation established
+    act: check installed WordPress themes
+    assert: All default themes should be installed
+    """
+    for unit_ip in unit_ip_list:
+        assert (
+                set(WordpressCharm._WORDPRESS_DEFAULT_THEMES) ==
+                set(get_theme_list_from_ip(unit_ip))
+        ), "default themes installed should match default themes defined in WordpressCharm"
+
+
+@pytest.mark.asyncio
+async def test_wordpress_install_uninstall_themes(
+        ops_test: pytest_operator.plugin.OpsTest,
+        application_name,
+        unit_ip_list,
+        get_theme_list_from_ip
+):
+    """
+    arrange: after WordPress charm has been deployed and db relation established
+    act: change themes setting in config
+    assert: themes should be installed and uninstalled accordingly
+    """
+    theme = "twentyfifteen"
+    application = ops_test.model.applications[application_name]
+    await application.set_config({"themes": theme})
+    await ops_test.model.wait_for_idle()
+
+    for unit_ip in unit_ip_list:
+        assert (
+                theme in get_theme_list_from_ip(unit_ip)
+        ), "theme should be installed after adding to the themes setting in config"
+
+    await application.set_config({"themes": ""})
+    await ops_test.model.wait_for_idle()
+
+    for unit_ip in unit_ip_list:
+        assert (
+                theme not in get_theme_list_from_ip(unit_ip)
+        ), "theme should be uninstalled after removing from the themes setting in config"
+
+
+@pytest.mark.asyncio
+async def test_wordpress_theme_installation_error(
+        ops_test: pytest_operator.plugin.OpsTest,
+        application_name
+):
+    invalid_theme = "sgkeahrgalejr"
```

Comment:
what about calling it some version of `invalid-theme-sgkeahrgalejr`? It can help with readability if it ends up in any log messages etc if it has at least a portion of it be meaningful

Reply:
Fixed, thanks!